app/helpers/blacklight/render_partials_helper_behavior.rb

```diff
@@ -22,6 +25,7 @@ def render_document_partials(doc, partials = [], locals = {})
       render_document_partial(doc, action_name, locals)
     end, "\n")
   end
+  deprecation_deprecate render_document_partials: 'Render the document_component instead'
```

Comment:
What does this mean? If I get this error I'm not sure what to do. Can we expand this deprecation error? Or if I just put in `render_document_component` instead of `render_document_partials` it'll be good?

Reply:
that would be a rather long message to stick here:

```
Replace this call with: "
document_component = blacklight_config.view_config(:atom).summary_component
render document_component.new(presenter: document_presenter(document), component: :div, show: true)" instead.
```

This assumes that the current context has a `document_presenter` helper and that `:atom` is the current config context.

Reply:
I might be in the minority here, but I'm :+1: to that long message. Changing it to "Replace this call with a document component render, e.g: allthat" might be nice to be more clear?